svc-bie-kafka/src/main/java/gov/va/vro/services/bie/config/MessageQueueConfig.java

```diff
@@ -0,0 +1,52 @@
+package gov.va.vro.services.bie.config;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.AbstractDeclarable;
+import org.springframework.amqp.core.Binding;
+import org.springframework.amqp.core.BindingBuilder;
+import org.springframework.amqp.core.Declarables;
+import org.springframework.amqp.core.FanoutExchange;
+import org.springframework.amqp.core.Queue;
+import org.springframework.amqp.support.converter.Jackson2JsonMessageConverter;
+import org.springframework.amqp.support.converter.MessageConverter;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Configuration
+public class MessageQueueConfig {
+
+  private static final boolean DURABLE = true;
```

Comment:
Why do you want to obfuscate these values?

Reply:
I prefer to do this with booleans because it makes it faster for other devs to understand whats going on without looking into what an unnamed true or false is doing. 

For example:
`final Queue queue = new Queue(topic, DURABLE)` is easier to interpret than `final Queue queue = new Queue(topic, true)` without digging into the implementation of the method call.

Reply:
when looking at `final Queue queue = new Queue(topic, DURABLE)` how would the reader know if the object created is durable or not?